### ISSUE TITLE
Can't import .rec files

### ISSUE THREAD

**Issue by alangrafu** [Freika/dawarich#384]
**Describe the bug**
I can't import .rec files from my owntracks sercver

**Version**
0.16.2

**To Reproduce**
1. Go to imports
2. Click on owntracks
3. Select a .rec file from owntracks server
4. Start import

**Expected behavior**
An error is reported that import failed

**Screenshots**

**Logs**
```
Import "2024-11.rec" failed: Connection refused - connect(2) for 127.0.0.1:6379 (redis://localhost:6379/1), stacktrace: /usr/local/lib/ruby/3.3.0/socket.rb:1219:in `__connect_nonblock' /usr/local/lib/ruby/3.3.0/socket.rb:1219:in `connect_nonblock' /usr/local/lib/ruby/3.3.0/socket.rb:60:in `connect_internal' /usr/local/lib/ruby/3.3.0/socket.rb:141:in `connect' /usr/local/lib/ruby/3.3.0/socket.rb:647:in `block in tcp' /usr/local/lib/ruby/3.3.0/socket.rb:231:in `each' /usr/local/lib/ruby/3.3.0/socket.rb:231:in `foreach' /usr/local/lib/ruby/3.3.0/socket.rb:637:in `tcp' /var/app/vendor/bundle/ruby/3.3.0/gems/redis-client-0.22.2/lib/redis_client/ruby_connection.rb:119:in `connect' /var/app/vendor/bundle/ruby/3.3.0/gems/redis-client-0.22.2/lib/redis_client/ruby_connection.rb:51:in `initialize' /var/app/vendor/bundle/ruby/3.3.0/gems/redis-client-0.22.2/lib/redis_client.rb:746:in `new' /var/app/vendor/bundle/ruby/3.3.0/gems/redis-client-0.22.2/lib/redis_client.rb:746:in `block in connect' /var/app/vendor/bundle/ruby/3.3.0/gems/redis-client-0.22.2/lib/redis_client/middlewares.rb:12:in `connect' /var/app/vendor/bundle/ruby/3.3.0/gems/redis-client-0.22.2/lib/redis_client.rb:745:in `connect' /var/app/vendor/bundle/ruby/3.3.0/gems/redis-client-0.22.2/lib/redis_client.rb:732:in `raw_connection' /var/app/vendor/bundle/ruby/3.3.0/gems/redis-client-0.22.2/lib/redis_client.rb:697:in `ensure_connected' /var/app/vendor/bundle/ruby/3.3.0/gems/redis-client-0.22.2/lib/redis_client.rb:292:in `call_v' /var/app/vendor/bundle/ruby/3.3.0/gems/redis-5.3.0/lib/redis/client.rb:90:in `call_v' /var/app/vendor/bundle/ruby/3.3.0/gems/redis-5.3.0/lib/redis.rb:152:in `block in send_command' /var/app/vendor/bundle/ruby/3.3.0/gems/redis-5.3.0/lib/redis.rb:151:in `synchronize' /var/app/vendor/bundle/ruby/3.3.0/gems/redis-5.3.0/lib/redis.rb:151:in `send_command' /var/app/vendor/bundle/ruby/3.3.0/gems/redis-5.3.0/lib/redis/commands/pubsub.rb:8:in `publish' /var/app/vendor/bundle/ruby/3.3.0/gems/actioncable-7.2.2/lib/action_cable/subscription_adapter/redis.rb:29:in `broadcast' /var/app/vendor/bundle/ruby/3.3.0/gems/actioncable-7.2.2/lib/action_cable/subscription_adapter/channel_prefix.rb:10:in `broadcast' /var/app/vendor/bundle/ruby/3.3.0/gems/actioncable-7.2.2/lib/action_cable/server/broadcasting.rb:56:in `block in broadcast' /var/app/vendor/bundle/ruby/3.3.0/gems/activesupport-7.2.2/lib/active_support/notifications.rb:210:in `block in instrument' /var/app/vendor/bundle/ruby/3.3.0/gems/activesupport-7.2.2/lib/active_support/notifications/instrumenter.rb:58:in `instrument' /var/app/vendor/bundle/ruby/3.3.0/gems/activesupport-7.2.2/lib/active_support/notifications.rb:210:in `instrument' /var/app/vendor/bundle/ruby/3.3.0/gems/actioncable-7.2.2/lib/action_cable/server/broadcasting.rb:54:in `broadcast' /var/app/vendor/bundle/ruby/3.3.0/gems/actioncable-7.2.2/lib/action_cable/server/broadcasting.rb:32:in `broadcast' /var/app/vendor/bundle/ruby/3.3.0/gems/actioncable-7.2.2/lib/action_cable/channel/broadcasting.rb:15:in `broadcast_to' /var/app/app/models/point.rb:45:in `broadcast_coordinates' /var/app/vendor/bundle/ruby/3.3.0/gems/activesupport-7.2.2/lib/active_support/callbacks.rb:362:in `block in make_lambda' /var/app/vendor/bundle/ruby/3.3.0/gems/activesupport-7.2.2/lib/active_support/callbacks.rb:208:in `call' /var/app/vendor/bundle/ruby/3.3.0/gems/activesupport-7.2.2/lib/active_support/callbacks.rb:563:in `block in invoke_after' /var/app/vendor/bundle/ruby/3.3.0/gems/activesupport-7.2.2/lib/active_support/callbacks.rb:563:in `each' /var/app/vendor/bundle/ruby/3.3.0/gems/activesupport-7.2.2/lib/active_support/callbacks.rb:563:in `invoke_after' /var/app/vendor/bundle/ruby/3.3.0/gems/activesupport-7.2.2/lib/active_support/callbacks.rb:111:in `run_callbacks' /var/app/vendor/bundle/ruby/3.3.0/gems/activesupport-7.2.2/lib/active_support/callbacks.rb:913:in `_run_commit_callbacks' /var/app/vendor/bundle/ruby/3.3.0/gems/activerecord-7.2.2/lib/active_record/transactions.rb:385:in `committed!' /var/app/vendor/bundle/ruby/3.3.0/gems/activerecord-7.2.2/lib/active_record/connection_adapters/abstract/transaction.rb:307:in `block in commit_records' /var/app/vendor/bundle/ruby/3.3.0/gems/activerecord-7.2.2/lib/active_record/connection_adapters/abstract/transaction.rb:344:in `run_action_on_records' /var/app/vendor/bundle/ruby/3.3.0/gems/activerecord-7.2.2/lib/active_record/connection_adapters/abstract/transaction.rb:306:in `commit_records' /var/app/vendor/bundle/ruby/3.3.0/gems/activerecord-7.2.2/lib/active_record/connection_adapters/abstract/transaction.rb:596:in `block in commit_transaction' /var/app/vendor/bundle/ruby/3.3.0/gems/activesupport-7.2.2/lib/active_support/concurrency/null_lock.rb:9:in `synchronize' /var/app/vendor/bundle/ruby/3.3.0/gems/activerecord-7.2.2/lib/active_record/connection_adapters/abstract/transaction.rb:584:in `commit_transaction' /var/app/vendor/bundle/ruby/3.3.0/gems/activerecord-7.2.2/lib/active_record/connection_adapters/abstract/transaction.rb:628:in `block in within_new_transaction' /var/app/vendor/bundle/ruby/3.3.0/gems/activesupport-7.2.2/lib/active_support/concurrency/null_lock.rb:9:in `synchronize' /var/app/vendor/bundle/ruby/3.3.0/gems/activerecord-7.2.2/lib/active_record/connection_adapters/abstract/transaction.rb:613:in `within_new_transaction' /var/app/vendor/bundle/ruby/3.3.0/gems/activerecord-7.2.2/lib/active_record/connection_adapters/abstract/database_statements.rb:361:in `transaction' /var/app/vendor/bundle/ruby/3.3.0/gems/activerecord-7.2.2/lib/active_record/transactions.rb:414:in `block in with_transaction_returning_status' /var/app/vendor/bundle/ruby/3.3.0/gems/activerecord-7.2.2/lib/active_record/connection_adapters/abstract/connection_pool.rb:421:in `with_connection' /var/app/vendor/bundle/ruby/3.3.0/gems/activerecord-7.2.2/lib/active_record/connection_handling.rb:296:in `with_connection' /var/app/vendor/bundle/ruby/3.3.0/gems/activerecord-7.2.2/lib/active_record/transactions.rb:410:in `with_transaction_returning_status' /var/app/vendor/bundle/ruby/3.3.0/gems/activerecord-7.2.2/lib/active_record/transactions.rb:362:in `save' /var/app/vendor/bundle/ruby/3.3.0/gems/activerecord-7.2.2/lib/active_record/suppressor.rb:52:in `save' /var/app/app/services/own_tracks/export_parser.rb:30:in `block in call' /var/app/app/services/own_tracks/export_parser.rb:17:in `each' /var/app/app/services/own_tracks/export_parser.rb:17:in `with_index' /var/app/app/services/own_tracks/export_parser.rb:17:in `call' /var/app/app/services/imports/create.rb:12:in `call' /var/app/app/models/import.rb:17:in `process!' /var/app/app/jobs/import_job.rb:10:in `perform' /var/app/vendor/bundle/ruby/3.3.0/gems/activejob-7.2.2/lib/active_job/execution.rb:68:in `block in _perform_job' /var/app/vendor/bundle/ruby/3.3.0/gems/activesupport-7.2.2/lib/active_support/callbacks.rb:121:in `block in run_callbacks' /var/app/vendor/bundle/ruby/3.3.0/gems/i18n-1.14.6/lib/i18n.rb:353:in `with_locale' /var/app/vendor/bundle/ruby/3.3.0/gems/activejob-7.2.2/lib/active_job/translation.rb:9:in `block (2 levels) in ' /var/app/vendor/bundle/ruby/3.3.0/gems/activesupport-7.2.2/lib/active_support/callbacks.rb:130:in `instance_exec' /var/app/vendor/bundle/ruby/3.3.0/gems/activesupport-7.2.2/lib/active_support/callbacks.rb:130:in `block in run_callbacks' /var/app/vendor/bundle/ruby/3.3.0/gems/activesupport-7.2.2/lib/active_support/core_ext/time/zones.rb:65:in `use_zone' /var/app/vendor/bundle/ruby/3.3.0/gems/activejob-7.2.2/lib/active_job/timezones.rb:9:in `block (2 levels) in ' /var/app/vendor/bundle/ruby/3.3.0/gems/activesupport-7.2.2/lib/active_support/callbacks.rb:130:in `instance_exec' /var/app/vendor/bundle/ruby/3.3.0/gems/activesupport-7.2.2/lib/active_support/callbacks.rb:130:in `block in run_callbacks' /var/app/vendor/bundle/ruby/3.3.0/gems/activesupport-7.2.2/lib/active_support/callbacks.rb:141:in `run_callbacks' /var/app/vendor/bundle/ruby/3.3.0/gems/activejob-7.2.2/lib/active_job/execution.rb:67:in `_perform_job' /var/app/vendor/bundle/ruby/3.3.0/gems/activejob-7.2.2/lib/active_job/instrumentation.rb:32:in `_perform_job' /var/app/vendor/bundle/ruby/3.3.0/gems/activejob-7.2.2/lib/active_job/execution.rb:51:in `perform_now' /var/app/vendor/bundle/ruby/3.3.0/gems/activejob-7.2.2/lib/active_job/instrumentation.rb:26:in `block in perform_now' /var/app/vendor/bundle/ruby/3.3.0/gems/activerecord-7.2.2/lib/active_record/railties/job_runtime.rb:13:in `block in instrument' /var/app/vendor/bundle/ruby/3.3.0/gems/activejob-7.2.2/lib/active_job/instrumentation.rb:40:in `block in instrument' /var/app/vendor/bundle/ruby/3.3.0/gems/activesupport-7.2.2/lib/active_support/notifications.rb:210:in `block in instrument' /var/app/vendor/bundle/ruby/3.3.0/gems/activesupport-7.2.2/lib/active_support/notifications/instrumenter.rb:58:in `instrument' /var/app/vendor/bundle/ruby/3.3.0/gems/activesupport-7.2.2/lib/active_support/notifications.rb:210:in `instrument' /var/app/vendor/bundle/ruby/3.3.0/gems/activejob-7.2.2/lib/active_job/instrumentation.rb:39:in `instrument' /var/app/vendor/bundle/ruby/3.3.0/gems/activerecord-7.2.2/lib/active_record/railties/job_runtime.rb:11:in `instrument' /var/app/vendor/bundle/ruby/3.3.0/gems/activejob-7.2.2/lib/active_job/instrumentation.rb:26:in `perform_now' /var/app/vendor/bundle/ruby/3.3.0/gems/activejob-7.2.2/lib/active_job/logging.rb:32:in `block in perform_now' /var/app/vendor/bundle/ruby/3.3.0/gems/activejob-7.2.2/lib/active_job/logging.rb:41:in `tag_logger' /var/app/vendor/bundle/ruby/3.3.0/gems/activejob-7.2.2/lib/active_job/logging.rb:32:in `perform_now' /var/app/vendor/bundle/ruby/3.3.0/gems/activejob-7.2.2/lib/active_job/execution.rb:29:in `block in execute' /var/app/vendor/bundle/ruby/3.3.0/gems/activesupport-7.2.2/lib/active_support/callbacks.rb:121:in `block in run_callbacks' /var/app/vendor/bundle/ruby/3.3.0/gems/activejob-7.2.2/lib/active_job/railtie.rb:79:in `block (4 levels) in ' /var/app/vendor/bundle/ruby/3.3.0/gems/activesupport-7.2.2/lib/active_support/reloader.rb:77:in `block in wrap' /var/app/vendor/bundle/ruby/3.3.0/gems/activesupport-7.2.2/lib/active_support/execution_wrapper.rb:87:in `wrap' /var/app/vendor/bundle/ruby/3.3.0/gems/activesupport-7.2.2/lib/active_support/reloader.rb:74:in `wrap' /var/app/vendor/bundle/ruby/3.3.0/gems/activejob-7.2.2/lib/active_job/railtie.rb:78:in `block (3 levels) in ' /var/app/vendor/bundle/ruby/3.3.0/gems/activesupport-7.2.2/lib/active_support/callbacks.rb:130:in `instance_exec' /var/app/vendor/bundle/ruby/3.3.0/gems/activesupport-7.2.2/lib/active_support/callbacks.rb:130:in `block in run_callbacks' /var/app/vendor/bundle/ruby/3.3.0/gems/activesupport-7.2.2/lib/active_support/callbacks.rb:141:in `run_callbacks' /var/app/vendor/bundle/ruby/3.3.0/gems/activejob-7.2.2/lib/active_job/execution.rb:27:in `execute' /var/app/vendor/bundle/ruby/3.3.0/gems/sidekiq-7.3.4/lib/active_job/queue_adapters/sidekiq_adapter.rb:10:in `perform' /var/app/vendor/bundle/ruby/3.3.0/gems/sidekiq-7.3.4/lib/sidekiq/processor.rb:220:in `execute_job' /var/app/vendor/bundle/ruby/3.3.0/gems/sidekiq-7.3.4/lib/sidekiq/processor.rb:185:in `block (4 levels) in process' /var/app/vendor/bundle/ruby/3.3.0/gems/sidekiq-7.3.4/lib/sidekiq/middleware/chain.rb:180:in `traverse' /var/app/vendor/bundle/ruby/3.3.0/gems/sidekiq-7.3.4/lib/sidekiq/middleware/chain.rb:183:in `block in traverse' /var/app/vendor/bundle/ruby/3.3.0/gems/sidekiq-7.3.4/lib/sidekiq/job/interrupt_handler.rb:9:in `call' /var/app/vendor/bundle/ruby/3.3.0/gems/sidekiq-7.3.4/lib/sidekiq/middleware/chain.rb:182:in `traverse' /var/app/vendor/bundle/ruby/3.3.0/gems/sidekiq-7.3.4/lib/sidekiq/middleware/chain.rb:183:in `block in traverse' /var/app/vendor/bundle/ruby/3.3.0/gems/sidekiq-7.3.4/lib/sidekiq/metrics/tracking.rb:26:in `track' /var/app/vendor/bundle/ruby/3.3.0/gems/sidekiq-7.3.4/lib/sidekiq/metrics/tracking.rb:134:in `call' /var/app/vendor/bundle/ruby/3.3.0/gems/sidekiq-7.3.4/lib/sidekiq/middleware/chain.rb:182:in `traverse' /var/app/vendor/bundle/ruby/3.3.0/gems/sidekiq-7.3.4/lib/sidekiq/middleware/chain.rb:173:in `invoke' /var/app/vendor/bundle/ruby/3.3.0/gems/sidekiq-7.3.4/lib/sidekiq/processor.rb:184:in `block (3 levels) in process' /var/app/vendor/bundle/ruby/3.3.0/gems/sidekiq-7.3.4/lib/sidekiq/processor.rb:145:in `block (6 levels) in dispatch' /var/app/vendor/bundle/ruby/3.3.0/gems/sidekiq-7.3.4/lib/sidekiq/job_retry.rb:118:in `local' /var/app/vendor/bundle/ruby/3.3.0/gems/sidekiq-7.3.4/lib/sidekiq/processor.rb:144:in `block (5 levels) in dispatch' /var/app/vendor/bundle/ruby/3.3.0/gems/sidekiq-7.3.4/lib/sidekiq/rails.rb:22:in `block in call' /var/app/vendor/bundle/ruby/3.3.0/gems/activesupport-7.2.2/lib/active_support/reloader.rb:77:in `block in wrap' /var/app/vendor/bundle/ruby/3.3.0/gems/activesupport-7.2.2/lib/active_support/execution_wrapper.rb:91:in `wrap' /var/app/vendor/bundle/ruby/3.3.0/gems/activesupport-7.2.2/lib/active_support/reloader.rb:74:in `wrap' /var/app/vendor/bundle/ruby/3.3.0/gems/sidekiq-7.3.4/lib/sidekiq/rails.rb:21:in `call' /var/app/vendor/bundle/ruby/3.3.0/gems/sidekiq-7.3.4/lib/sidekiq/processor.rb:139:in `block (4 levels) in dispatch' /var/app/vendor/bundle/ruby/3.3.0/gems/sidekiq-7.3.4/lib/sidekiq/processor.rb:281:in `stats' /var/app/vendor/bundle/ruby/3.3.0/gems/sidekiq-7.3.4/lib/sidekiq/processor.rb:134:in `block (3 levels) in dispatch' /var/app/vendor/bundle/ruby/3.3.0/gems/sidekiq-7.3.4/lib/sidekiq/job_logger.rb:23:in `call' /var/app/vendor/bundle/ruby/3.3.0/gems/sidekiq-7.3.4/lib/sidekiq/processor.rb:133:in `block (2 levels) in dispatch' /var/app/vendor/bundle/ruby/3.3.0/gems/sidekiq-7.3.4/lib/sidekiq/job_retry.rb:85:in `global' /var/app/vendor/bundle/ruby/3.3.0/gems/sidekiq-7.3.4/lib/sidekiq/processor.rb:132:in `block in dispatch' /var/app/vendor/bundle/ruby/3.3.0/gems/sidekiq-7.3.4/lib/sidekiq/job_logger.rb:50:in `prepare' /var/app/vendor/bundle/ruby/3.3.0/gems/sidekiq-7.3.4/lib/sidekiq/processor.rb:131:in `dispatch' /var/app/vendor/bundle/ruby/3.3.0/gems/sidekiq-7.3.4/lib/sidekiq/processor.rb:183:in `block (2 levels) in process' /var/app/vendor/bundle/ruby/3.3.0/gems/sidekiq-7.3.4/lib/sidekiq/processor.rb:182:in `handle_interrupt' /var/app/vendor/bundle/ruby/3.3.0/gems/sidekiq-7.3.4/lib/sidekiq/processor.rb:182:in `block in process' /var/app/vendor/bundle/ruby/3.3.0/gems/sidekiq-7.3.4/lib/sidekiq/processor.rb:181:in `handle_interrupt' /var/app/vendor/bundle/ruby/3.3.0/gems/sidekiq-7.3.4/lib/sidekiq/processor.rb:181:in `process' /var/app/vendor/bundle/ruby/3.3.0/gems/sidekiq-7.3.4/lib/sidekiq/processor.rb:86:in `process_one' /var/app/vendor/bundle/ruby/3.3.0/gems/sidekiq-7.3.4/lib/sidekiq/processor.rb:76:in `run' /var/app/vendor/bundle/ruby/3.3.0/gems/sidekiq-7.3.4/lib/sidekiq/component.rb:10:in `watchdog' /var/app/vendor/bundle/ruby/3.3.0/gems/sidekiq-7.3.4/lib/sidekiq/component.rb:19:in `block in safe_thread' 
```

**Additional context**

**Comment by alangrafu** [comment]
The only change in my docker compose was the change of the dawarich app port to

3432:3000

**Comment by Freika** [comment]
@alangrafu it should be fixed in 0.16.3, update your instance and try again

**Comment by alangrafu** [comment]
It works now, thanks!
